### DISCUSSION TITLE
[HDL] Design Decisions in LiveHD for HDLs Compilation

### DISCUSSION THREAD

**Discussion by swang203** [cucapra/latte21#15] [Papers]
Author(s): Sheng-Hong Wang and Jose Renau
Paper PDF: https://capra.cs.cornell.edu/latte21/paper/28.pdf
Talk Recording: https://youtu.be/oKjYDUqeSoU

**Comment by rachitnigam** [comment]
@swang203 Your YouTube talk is marked as "content for kids" which means we can't add it to our playlist. Can you update the setting for the video and mark it not for kids? Please ping me (by saying @rachitnigam) when you're done.

**Reply by swang203** [reply]
@rachitnigam
